configure.py

```diff
@@ -7,28 +7,28 @@
-parser.add_option("--qt-dir", action="store", type="string", dest="qt-dir", default="", help="qt (qmake) directory")
-parser.add_option("--qt-dir-xp", action="store", type="string", dest="qt-dir-xp", default="", help="qt for Windows XP (qmake) directory")
-parser.add_option("--compiler", action="store", type="string", dest="compiler", default="", help="compiler name")
-parser.add_option("--no-apps", action="store", type="string", dest="no-apps", default="0", help="disable build desktop apps (qt)")
-parser.add_option("--themesparams", action="store", type="string", dest="themesparams", default="", help="presentation themes thumbnails additions")
-parser.add_option("--git-protocol", action="store", type="string", dest="git-protocol", default="https", help="presentation themes thumbnails additions")
-parser.add_option("--branding", action="store", type="string", dest="branding", default="", help="branding path")
-parser.add_option("--branding-name", action="store", type="string", dest="branding-name", default="", help="branding name")
-parser.add_option("--branding-url", action="store", type="string", dest="branding-url", default="", help="branding url")
-parser.add_option("--sdkjs-addon", action="append", type="string", dest="sdkjs-addons", default=[], help="sdkjs addons")
-parser.add_option("--sdkjs-addon-desktop", action="append", type="string", dest="sdkjs-addons-desktop", default=[], help="sdkjs addons desktop")
-parser.add_option("--server-addon", action="append", type="string", dest="server-addons", default=[], help="server addons")
-parser.add_option("--web-apps-addon", action="append", type="string", dest="web-apps-addons", default=[], help="web-apps addons")
-parser.add_option("--sdkjs-plugin", action="append", type="string", dest="sdkjs-plugin", default=["default"], help="sdkjs addons for all products")
-parser.add_option("--sdkjs-plugin-server", action="append", type="string", dest="sdkjs-plugin-server", default=["default"], help="sdkjs addons for server")
+parser.add_option("--update", action="store", type="string", dest="update", default="1", help="defines whether it's necessary to update/clone repos. If it's set to true, build_tools automatically get the necessary subrepos. If it's set to false, you should define which ones to use")
+parser.add_option("--update-light", action="store", type="string", dest="update-light", default="", help="performs pull/clone without switching branches, can be used only if  update is true.")
+parser.add_option("--branch", action="store", type="string", dest="branch", default="master", help="branch/tag name, used only if update is true and update_light is not used. Updates/clones all the repos and switches the branch to the proper one deleting all the local changes")
+parser.add_option("--clean", action="store", type="string", dest="clean", default="1", help="defines whether to build everything anew (or changed  c/c++ sources only).")
```

Comment:
Do not fully understand meaning of `(or changed  c/c++ sources only)`
In general I understand meaning of this option, but this note is stange

Reply:
If it's clear in general, but this note is confusing, I'm deleting it